cmd/clusterctl/cmd/init.go

```diff
@@ -83,9 +83,9 @@ var initCmd = &cobra.Command{
 
 func init() {
 	initCmd.Flags().StringVarP(&io.kubeconfig, "kubeconfig", "", "", "Path to the kubeconfig file to use for accessing the management cluster. If empty, default rules for kubeconfig discovery will be used")
-	initCmd.Flags().StringVarP(&io.coreProvider, "core", "", "", "Infrastructure providers to add to the management cluster. By default (empty), the cluster-api core provider is installed on the first init")
-	initCmd.Flags().StringSliceVarP(&io.infrastructureProviders, "infrastructure", "i", nil, "Infrastructure providers to add to the management cluster")
-	initCmd.Flags().StringSliceVarP(&io.bootstrapProviders, "bootstrap", "b", nil, "Bootstrap providers to add to the management cluster. By default (empty), the kubeadm bootstrap provider is installed on the first init")
+	initCmd.Flags().StringVarP(&io.coreProvider, "core", "", "", "Core provider version (by tag) to add to the management cluster. By default (empty), the cluster-api core provider's latest release is used")
+	initCmd.Flags().StringSliceVarP(&io.infrastructureProviders, "infrastructure", "i", nil, "Infrastructure providers and versions (by tag) to add to the management cluster")
+	initCmd.Flags().StringSliceVarP(&io.bootstrapProviders, "bootstrap", "b", nil, "Bootstrap providers and versions (by tag) to add to the management cluster. By default (empty), the kubeadm bootstrap provider's latest release is used")
```

Comment:
Maybe provide an example instead of `(by tag)` ?

Reply:
no problem, probably a good idea. How about:

Bootstrap providers and versions, for example "aws:v0.4.1" to add to the management cluster. By default (empty), the kubeadm bootstrap provider's latest release is used"

and add to the others as well to get right of all the (by tag)

Reply:
> for example "aws:v0.4.1"

(e.g. `aws:v0.4.1`)

Reply:
updated!   note I used the unreleased version numbers because due to naming of component files, clusterctl v2 won't work with earlier releases